apis/v1alpha1/httproute_types.go

```diff
@@ -377,10 +378,9 @@ type HTTPRequestHeaderFilter struct {
 	//   GET /foo HTTP/1.1
 	//   My-Header2: DEF
 	//
-	// Support: extended?
+	// Support: Extended
+	// +kubebuilder:validation:MaxItems=16
```

Comment:
Why is remove limited to 16 but add is not?

Reply:
Sadly I can't find a way to limit the number of keys in a map with kubebuilder tags (or CRD validation in general).

Reply:
`// +kubebuilder:validation:MinItems=1`  to match other requried lists

Reply:
@danehans I don't think we want to have a minimum here? Wouldn't it be valid to only add headers and not remove headers with this filter?